imapclient/imapclient.py

```diff
@@ -1166,6 +1169,16 @@ def copy(self, messages, folder):
                                        self._normalise_folder(folder),
                                        uid=True, unpack=True)
 
+    def move(self, messages, folder):
+        """Atomically move messages to another folder.
+
+        Requires the MOVE capability, see :rfc:`6851`.
+        """
```

Comment:
It might be worth mentioning that `messages` must contain UIDs, regardless of what `use_uid` is set to.

Reply:
Done